app/api/v1/airline/[airlineId]/route.ts

```diff
@@ -0,0 +1,316 @@
+import { NextRequest, NextResponse } from "next/server"
+
+import { getDatabase } from "@/lib/couchbase-connection"
+import { TAirline,AirlineSchema } from "@/app/models/Airline"
+import { DocumentNotFoundError,DocumentExistsError } from 'couchbase';
+
+import { ZodError } from "zod";
+
+/**
+ * @swagger
+ * /api/v1/airline/{airlineId}:
+ *   get:
+ *     summary: Get an airline by ID
+ *     description: |
+ *       Get Airline with specified ID.
+ *       
+ *       This provides an example of using [Key Value operations](https://docs.couchbase.com/nodejs-sdk/current/howtos/kv-operations.html) in Couchbase to get a document with specified ID.
+ *       
+ *       Key Value operations are unique to Couchbase and provide very high-speed get/set/delete operations.
+ *       
+ *       Code: `airline/[airlineId]/route.ts` Method: `GET`
+ *     tags:
+ *        - Airline
+ *     parameters:
+ *       - name: airlineId
+ *         in: path
+ *         description: ID of the airline
```

Comment:
Can you please update this to: Airline ID like airline_10

Reply:
updated in all

Reply:
![image](https://github.com/couchbase-examples/nextjs-capella-quickstart/assets/44238657/6f656651-efe4-4220-8e4e-64ab37e4d284)

Reply:
added examples in all possible routes